### ISSUE TITLE
Use JSON in HTML options

### ISSUE THREAD

**Issue by phucbm** [viivue/easy-popup#7]
It should be able to set the options via HTML JSON like this:

```html
<!-- Init with options -->
<select data-easy-select='{ "id":"my-id", "align":"right" }'>
    <option value="1">One</option>
</select>
```

Instead of the current style:

```html
<div data-easy-popup="demo"
     data-easy-popup-mobile="true"
     data-easy-popup-title="Popup title"
     data-easy-popup-theme="my-style"
     data-easy-popup-click-outside-to-close="false">
    <p>Your content.</p>
</div>
```

**Comment by phucbm** [comment]
@vuquangpham 

Could you please check what we're doing in Easy Select to apply to this repo? Thank you.

**Comment by phucbm** [comment]
Closed with #8
